hypertrace-core-graphql-export-span-utils/src/main/java/org/hypertrace/core/graphql/utils/export/span/ExportSpanResponse.java

```diff
@@ -0,0 +1,57 @@
+package org.hypertrace.core.graphql.utils.export.span;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.protobuf.Message;
+import com.google.protobuf.util.JsonFormat;
+import io.opentelemetry.proto.trace.v1.ResourceSpans;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.experimental.Accessors;
+import org.apache.commons.text.StringEscapeUtils;
+
+@lombok.Value
+@Accessors(fluent = true)
+public class ExportSpanResponse {
```

Comment:
We have an ExportSpan, an ExportSpanResponse and an ExportSpanResult - it's not clear to me what their roles are, perhaps a rename is in onder?

Reply:
`ExportSpan` is a wrapper around otlp `ResourceSpans` structure. `ExportSpanResult` is GraphQL API string result holder. Removed the `ExportSpanResponse`